internal/repository/sync.go

```diff
@@ -64,8 +68,49 @@ func Clone(ctx context.Context, remoteURL, dir, initialBranch string) (*Reposito
 		return nil, errors.Join(ErrCloningRepository, err)
 	}
 
+	sort.Slice(expectedRootKeys, func(i, j int) bool {
+		return expectedRootKeys[i].KeyID < expectedRootKeys[j].KeyID
+	})
+
 	repository := &Repository{r: r}
 
 	slog.Debug("Verifying HEAD...")
+	if len(expectedRootKeys) > 0 {
+		slog.Debug("Getting root keys...")
```

Comment:
```suggestion
		slog.Debug("Loading embedded root keys...")
```

Reply:
We want to be clear which set of root keys we mean in our debug statements.

Reply:
Fixed